src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseworkerIssueFinalDecision.java

```diff
@@ -154,10 +163,44 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
 
     public SubmittedCallbackResponse submitted(CaseDetails<CaseData, State> details,
                                                CaseDetails<CaseData, State> beforeDetails) {
-        var cicCase = details.getData().getCicCase();
+        var data = details.getData();
+        var cicCase = data.getCicCase();
+        String caseNumber = data.getHyphenatedCaseRef();
+
+        Document finalDecisionGuidance = getFinalDecisionGuidanceDocument(data, details.getId());
+        data.getCaseIssueFinalDecision().setFinalDecisionGuidance(finalDecisionGuidance);
+
+        final StringBuilder messageLine2 = new StringBuilder(100);
+        messageLine2.append(" A notification will be sent  to: ");
```

Comment:
Why 100? 
If that's a String length limit, make it a constant field...

Reply:
Removed the String limit as its not needed. Used default constructor instead